File: mobile/src/main/java/com/pdyjak/powerampwear/BootUpReceiver.java
package com.pdyjak.powerampwear;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BootUpReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent i) {
        context.startService(new Intent(context, BackgroundService.class));
    }
}

File: mobile/src/main/java/com/pdyjak/powerampwear/BootUpReceiver.kt
package com.pdyjak.powerampwear

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootUpReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action != Intent.ACTION_BOOT_COMPLETED) return
        context.startService(Intent(context, BackgroundService::class.java))
    }
}
